### PR TITLE
Lesson05 figma

### PR DESCRIPTION
**Pull request by siefimov**
Please review

### CODE REVIEW

File: homework05-figma/index.html [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+               </article>
+            </section>
+            <br />
+            <br />
+
+            <!-- PORTFOLIO -->
+            <section>
+               <h2 style="text-align: center">Portfolio</h2>
+               <article>
+                  <img src="./assets/image/fashion-store.png" alt="" />
```
Comment (marynabudz1) [inline homework05-figma/index.html:250]: provide alt for img

File: homework05-figma/index.html [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+                  <p style="text-align: center">
+                     Online fashion store - <br />
+                     Homepage
+                  </p>
+               </article>
+               <br />
+               <br />
+               <article>
+                  <img src="./assets/image/reebok_store.png" alt="" />
```
Comment (marynabudz1) [inline homework05-figma/index.html:259]: provide alt for img

File: homework05-figma/index.html [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+                  <p style="text-align: center">Reebok Store - Concept</p>
+               </article>
+               <br />
+               <br />
+               <article>
+                  <img src="./assets/image/braun-landing.png" alt="" />
```
Comment (marynabudz1) [inline homework05-figma/index.html:265]: provide alt for img

File: homework05-figma/index.html [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+                  <button type="submit">Send message</button>
+                  <br />
+                  <br />
+                  <div
+                     style="display: flex; gap: 50px; justify-content: center"
+                  >
+                     <a href="#" target="_blank"
+                        ><img
+                           src="./assets/image/linkedin.svg"
+                           alt="Instagram logo"
```
Comment (marynabudz1) [inline homework05-figma/index.html:292]: chage alt for img

File: homework05-figma/index.html [hunk 5]
```diff
@@ -0,0 +1,325 @@
...
+                     /></a>
+                     <a href="#" target="_blank"
+                        ><img
+                           src="./assets/image/instagram.svg"
+                           alt="Instagram logo"
+                     /></a>
+                     <a href="#" target="_blank"
+                        ><img
+                           src="./assets/image/behance.svg"
+                           alt="Instagram logo"
```
Comment (marynabudz1) [inline homework05-figma/index.html:302]: chage alt for img

File: homework05-figma/index.html [hunk 6]
```diff
@@ -0,0 +1,325 @@
...
+                     /></a>
+                     <a href="#" target="_blank"
+                        ><img
+                           src="./assets/image/dribbble.svg"
+                           alt="Instagram logo"
```
Comment (marynabudz1) [inline homework05-figma/index.html:307]: chage alt for img

**Review by marynabudz1 — APPROVED** [review]

File: homework05-figma/index.html (hunk 1 above)
Comment (siefimov) [reply homework05-figma/index.html:250]: done

File: homework05-figma/index.html (hunk 2 above)
Comment (siefimov) [reply homework05-figma/index.html:259]: done

File: homework05-figma/index.html (hunk 3 above)
Comment (siefimov) [reply homework05-figma/index.html:265]: done

File: homework05-figma/index.html (hunk 4 above)
Comment (siefimov) [reply homework05-figma/index.html:292]: done

File: homework05-figma/index.html (hunk 5 above)
Comment (siefimov) [reply homework05-figma/index.html:302]: done

Comment (siefimov) [reply homework05-figma/index.html:302]: done

Comment (siefimov) [reply homework05-figma/index.html:302]: done

File: homework05-figma/index.html (hunk 6 above)
Comment (siefimov) [reply homework05-figma/index.html:307]: done

**Pull request merged**